organization/articles-of-association.es.md

```diff
@@ -0,0 +1,455 @@
+> El presente documento contienen una traducción no oficial desarrollada únicamente con fines informativos. El texto vinculante, por tanto, será el del [documento holandés original](https://github.com/publiccodenet/about/blob/develop/organization/articles-of-association.nl.md) en todos los aspectos legales.
+
+# Artículos de la asociación 'Foundation for Public Code Vereniging'
+
+El 3 de junio de dos mil diecinueve, el Sr. Lourens Wouter de Waard, notario con sede en Ámsterdam, comparece ante mí:
+
+Doña Ellen Tine Vermeulen, abogada, empleada en las oficinas de la empresa privada de responsabilidad limitada: Arcagna B.V., que tiene su domicilio social en Ámsterdam y con dirección: 1071 DJ Amsterdam, Museumplein 5 E&F, nacida en Ede el veintidós de agosto de mil novecientos setenta y dos, actuando en esta capacidad como apoderada escrita de:
+
+Boris Hendrik van Hoytema (datos eliminados deliberadamente) (el "Fundador 1");
```

Comment:
This is a numbered list in the other versions.

Reply:
Done! 👍